### PR TITLE
Fix configure file parse

### PR DESCRIPTION
**Pull request by hustcat**
Fix configure file parse

### CODE REVIEW

**Comment by mrd0ll4r** [comment]
I noticed this didn't parse correctly too.

However, I'd like to keep the config file as is, i.e. not have the `middleware` as a block just to hold the `announce_interval`.

There is an `inline` annotation, i.e.

```go
type A struct {
    B `yaml:",inline"`
    D int `yaml:"d"`
}

type B struct {
    C int `yaml:"c"`
}
```

File: cmd/chihaya/config.go [hunk 1]
```diff
@@ -23,13 +23,13 @@ type hookConfig struct {
...
 type ConfigFile struct {
 	MainConfigBlock struct {
-		middleware.Config
-		PrometheusAddr string              `yaml:"prometheus_addr"`
-		HTTPConfig     httpfrontend.Config `yaml:"http"`
-		UDPConfig      udpfrontend.Config  `yaml:"udp"`
-		Storage        memory.Config       `yaml:"storage"`
-		PreHooks       []hookConfig        `yaml:"prehooks"`
-		PostHooks      []hookConfig        `yaml:"posthooks"`
+		middleware.Config `yaml:"middleware"`
```
Comment (mrd0ll4r) [inline cmd/chihaya/config.go]: Change this to use the `inline` annotation, please.

**Comment by hustcat** [comment]
@mrd0ll4r Thank you. I have changed it:)

**Comment by mrd0ll4r** [comment]
LGTM, thanks for the PR! :)

**Pull request merged**